metricbeat/docs/fields.asciidoc

```diff
@@ -42645,16 +42645,6 @@ Kubernetes API server metrics
 
 
 
-*`kubernetes.apiserver.request.client`*::
```

Comment:
Add a note in the PR's description that it's not a breaking change but a bug fix instead to avoid future confusions.